### PR TITLE
Admin merchant dashboard#28

### LINKED ISSUE #28: User Story 43, Admin can see a merchant's dashboard
```
As an admin user
When I visit the merchant index page ("/merchants")
And I click on a merchant's name,
Then my URI route should be ("/admin/merchants/6")
Then I see everything that merchant would see
```

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #33

child of #79

### PR DESCRIPTION
**Pull request by timnallen**
As an admin user
When I visit the merchant index page ("/merchants")
And I click on a merchant's name,
Then my URI route should be ("/admin/merchants/6")
Then I see everything that merchant would see

closes #28

### CODE REVIEW

File: app/controllers/admin/items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Admin::ItemsController < Admin::BaseController
+  def index
+    @items = Item.where(user_id: params[:merchant_id])
+    render :'merchant/items/index'
```
Comment (teresa-m-knowles) [inline app/controllers/admin/items_controller.rb:4]: great workaround

**Review by teresa-m-knowles — APPROVED** [review]
Great work, Tim

File: app/controllers/admin/items_controller.rb (hunk 1 above)
Comment (csvlewis) [reply app/controllers/admin/items_controller.rb:4]: I agree, way to reuse code with the render!

**Review by csvlewis — APPROVED** [review]

**Review by allieIsOkay — APPROVED** [review]
